lib/global_uid/migration_extension.rb

```diff
@@ -30,7 +30,7 @@ def create_table(name, options = {}, &blk)
     end
 
     def drop_table(name, options = {})
-      if !GlobalUid.configuration.disabled? && options[:use_global_uid] == true
+      if GlobalUid.enabled? && options[:use_global_uid] == true
```

Comment:
I'm pretty sure this should be changed to `options[:use_global_uid] != false` as it's implicit behaviour, clients need to opt out. Like we're doing above 
https://github.com/zendesk/global_uid/blob/6705d48af1f4accf5d112b9f43c8889c5abc5753/lib/global_uid/migration_extension.rb#L5-L6

It's been like that for a long time though, so I may be wrong.

Reply:
So if `options[:use_global_uid]` is `nil`, we’ll create the tables but we won’t drop them. 🤷

Reply:
Yep.. planning on leaving that change for another day. I foresee side effects :)